django_project/core/base_templates/index.html

```diff
@@ -40,6 +40,7 @@
 	{% block side_navbar %}
 		{% include 'sidebar.html' %} <!-- side_navbar.html -->
 	{% endblock %}
+	<div class='body-wrapper'>
```

Comment:
could you add this class to `.container` div? 
What's the benefit of adding a new div?

Reply:
It needs to be outside the block body - what is inside the block body gets replaced and is not relevant.